src/app/payment/storedInstrument/ManageInstrumentsModal.tsx

```diff
@@ -86,7 +86,7 @@ class ManageInstrumentsModal extends Component<ManageInstrumentsModalProps & Wit
         const accountInstruments = instruments.filter(isAccountInstrument);
 
         return (
-            accountInstruments.length
+            accountInstruments.length || bankInstruments.length
```

Comment:
As your condition statement and block both deal with the same array
we can improve legibility and help keep them both in sync by sharing a singular reference to that array.

E.g.

```
const bankAndAccountInstruments = [...bankInstruments, ...accountInstruments];

return (
    bankAndAccountInstruments.length
    ? <ManageAccountInstrumentsTable instruments={bankAndAccountInstruments}
```

Reply:
Ternary operators also make things hard to read (especially given the size of JSX within this one)

It's probably worth considering multiple returns instead.

E.g.

```
if (bankAndAccountInstruments.length) {
    return <ManageAccountInstrumentsTable \\ etc...
}

return <ManageCardInstrumentsTable \\ etc...
```

Reply:
Changes were made based on this comments.